apps/charterafrica/src/components/Documents/DocumentCard.js

```diff
@@ -36,6 +42,15 @@ const DocumentCard = React.forwardRef(function DocumentCard(props, ref) {
             background: "transparent",
           },
         }}
+        component={Link}
+        href={{
+          pathname: `${asPath}/documents/`,
+          query: {
+            url: canonicalUrl,
+            title,
+            ...options,
+          },
```

Comment:
Shouldn't this all be done in `processPageConsultation` (similar to how we compute href for articles)?

Reply:
@kilemensi This won't work because we fetch the document details using the [API](https://github.com/CodeForAfrica/ui/blob/54d28b22451b3ba032fdfb11d4d15b8ef9970ab7/apps/charterafrica/src/components/Documents/Documents.js#L20-L24) then pass it as props to the `DocumentCard`. We do not use `processPageConsultation`.